packages/phone-number-privacy/combiner/src/common/input-validation.ts

```diff
@@ -25,6 +25,11 @@ export function hasValidPhoneNumberHash(requestBody: any): boolean {
   return requestBody.hashedPhoneNumber && isByte32(requestBody.hashedPhoneNumber)
 }
 
+export function hasValidTimestamp(requestBody: any): boolean {
```

Comment:
You can add this to the Common library so that we don't have to duplicate it across the combiner/signer:
https://github.com/celo-org/celo-monorepo/blob/master/packages/phone-number-privacy/combiner/src/common/input-validation.ts